api/v1_prizes_claim.go

```diff
@@ -0,0 +1,451 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	mathrand "math/rand"
+	"time"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgtype"
+	"go.uber.org/zap"
+)
+
+const (
+	yakMintAddress       = "ZDaUDL4XFdEct7UgeztrFQAptsvh4ZdhyZDZ1RpxYAK"
+	yakClaimAmount       = 2000000000 // 2 YAK with 9 decimals - amount required to claim a prize
+	yakAirdropAmount     = 1000000000 // 1 YAK with 9 decimals - amount awarded in coin airdrop prizes
+	prizeReceiverAddress = "EHd892m3xNWGBuAXnafavqcFjXXUZp9bGecdSDNP2SLR"
```

Comment:
what address is this?

Reply:
This is just a random wallet I own for testing purposes